packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -1556,3 +1556,13 @@ class DebounceOperationQueue<T, K> {
     return completer.future;
   }
 }
+
+/// Specialized exception for returning errors to the daemon client.
+class DaemonError implements Exception {
```

Comment:
IIRC we try to make it clear that a class is an `Exception` in the name so that it will be obvious that it will be caught by catch blocks that handle the `Exception` type.

```suggestion
class DaemonException implements Exception {
```

Reply:
+1

Reply:
Done